haystack/components/audio/whisper_local.py

```diff
@@ -4,15 +4,12 @@
 import tempfile
 from pathlib import Path
 
-from haystack import component, Document, default_to_dict, ComponentError
+from haystack import component, Document, default_to_dict, ComponentError, default_from_dict
 from haystack.dataclasses import ByteStream
 from haystack.lazy_imports import LazyImport
+from haystack.utils import ComponentDevice
 
-with LazyImport(
-    "Run 'pip install transformers[torch]' to install torch and "
-    "'pip install \"openai-whisper>=20231106\"' to install whisper."
-) as whisper_import:
-    import torch
```

Comment:
- [`torch` is a fundamental requirement for `whisper`](https://github.com/openai/whisper/blob/main/requirements.txt)
- `torch` import is no longer used
so I removed it